docs/components/webview.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebarDepth: 0
+---
+
+# WebView
+
+`<WebView>` is a UI component that lets you show web content in your app. You can pull and show content from a URL or a local HTML file, or you can render static HTML content.
```

Comment:
I can not see live demo section, please use documentation template in /docs/components/_template.md

Reply:
I have added live demo heading to appropriate place with a new commit.